indico/modules/core/templates/impersonation.html

```diff
@@ -22,6 +22,14 @@
                 title="{% trans user=message %}Switch back to {{ original_name }}{% endtrans %}"></i>
         </div>
     </div>
+    <script>
+        (() => {
+            let numBars = Array.from(document.querySelectorAll('.announcement-bar'))
+                .filter(function(e) {return e.style.display === 'flex';})
```

Comment:
Isn't it possible to use arrow functions here?

Reply:
It is. We don't support browsers anymore that do not support arrow functions.